app/mailers/planning_application_mailer.rb

```diff
@@ -44,18 +55,19 @@ def receipt_notice_mail(planning_application, host)
     view_mail(
       NOTIFY_TEMPLATE_ID,
       subject: "We have received your application",
-      to: @planning_application.applicant_email,
+      to: email_applicant_and_agent(@planning_application),
     )
   end
 
   def validation_request_mail(planning_application, validation_request)
     @planning_application = planning_application
     @validation_request = validation_request
+    @application_accountable_email = @planning_application.agent_email.presence || @planning_application.applicant_email
```

Comment:
why is this not using the method you made?

Reply:
this one is the exception - in all other cases we want to email both, but for "validation requests" (where an action is required) we want to only email agent if an agent is present, or email the applicant if no agent is present

Reply:
OK; for what it's worth one option could have been `applicant_and_agent_email.first(&:present?)` but happy with it anyway